rel/overlay/etc/default.ini

```diff
@@ -578,3 +584,44 @@ compaction = false
 ; CouchDB will use the value of `max_limit` instead. If neither is
 ; defined, the default is 2000 as stated here.
 ; max_limit_partitions = 2000
+
+[reshard]
+; Maximum resharding jobs allowed. New jobs cannot be added until some
+; old ones are deleted. A finished job is not automatically
+; removed. It has to be removed by hand.
+;max_jobs = 48
+;
+; Time to wait before retrying a failed shard splitting phase.
```

Comment:
Realising that the docs here vs. https://github.com/apache/couchdb-documentation/blob/a35ea115a71fd27a21245a6f14becf43adcfc8a7/src/config/resharding.rst are not in sync, can you make the text identical in each place? Sometimes your new description here is better, sometimes the one in the docs repo is better.

Reply:
Another good catch, the main  docs are outdated, will make a PR to update them

Reply:
Fixed in this documentation PR:

https://github.com/apache/couchdb-documentation/pull/478

I had also opted to keep the description in the documentation to avoid redundancy

Reply:
docs PR merged upstream